packages/firebase_ml_vision/pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dev_dependencies:
   image_picker: ^0.4.5
   flutter_test:
     sdk: flutter
+  firebase_core: ^0.2.5+1
```

Comment:
I think you forgot to update the pubspec version.

Reply:
done, thanks